src/main/java/com/synopsys/integration/alert/workflow/startup/component/AlertStartupInitializer.java

```diff
@@ -56,7 +56,7 @@
 import com.synopsys.integration.alert.web.config.FieldModelProcessor;
 
 @Component
-@Order(1)
+@Order(10)
```

Comment:
Will things with order always get initialized before things without order?

Reply:
Components without the Order annotation will be assigned the lowest priority and get initialized after the components with the Order annotation.
From the Spring docs:
https://docs.spring.io/spring/docs/5.0.1.RELEASE/javadoc-api/org/springframework/core/OrderComparator.html